packages/ember-engines/addon/-private/engine-ext.js

```diff
@@ -1,16 +0,0 @@
-import Application from '@ember/application';
-import Engine from '@ember/engine';
-
-import ExternalLinkComponent from '../components/link-to-external';
-
-Engine.reopen({
-  buildRegistry() {
-    let registry = this._super(...arguments);
-
-    if (!(this instanceof Application)) {
-      registry.register('component:link-to-external', ExternalLinkComponent);
```

Comment:
@void-mAlex any specific reason we can't keep this and do not require engine users to re-export `link-to-external` themselves?

Reply:
people shouldn't need to re-export it themselves as it's provided by an app re-export within ember-engines itself
this just makes the system a bit saner and not assume things about the registry existing and being the thing that is used to resolve components (gets replaced under embrodier staticComponents with compile time lookups)